src/generic/eigen_solver.h

```diff
@@ -167,6 +173,14 @@ namespace oomph
                             Vector<std::complex<double>>& eigenvalue,
                             Vector<DoubleVector>& eigenvector);
 
+    void solve_adjoint_eigenproblem(Problem* const& problem_pt,
+                                    const int& n_eval,
+                                    Vector<std::complex<double>>& eigenvalue,
+                                    Vector<DoubleVector>& eigenvector)
+    {
+      BrokenCopy::broken_assign("ARPACK");
+    }
```

Comment:
Replacing
```
    {
      BrokenCopy::broken_assign("ARPACK");
    }
```
with
```suggestion
    = delete;
```
Results in an error. "deleted function '...' overriding non-deleted function. Is this because it is 'implementing' a pure virtual method with a deleted function? Suggestions on how to fix it @puneetmatharu? Thanks

Reply:
Ah, interesting! The error arises because the base class `EigenSolver` has a non-broken implementation of `solve_adjoint_eigenproblem(...)` and deleting the implementation in a derived class would screw up uses of polymorphism, so the compiler (rightly) complains. 

Given that the function is pure virtual in the base-class, it should be overridden in *some* base class (if not `ARPACK` or `LAPACK_QZ`). Therefore, it doesn't actually make sense to actually `delete` it. My mistake - apologies!